test/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM centos:centos7
+
+LABEL name="EL7 OSG 3.5 Gratia Probe Test Container"
+
+RUN yum install -y yum-priorities \
+    && yum install -y \
+       http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm \
+    && yum install -y \
+       http://repo.opensciencegrid.org/osg/3.5/osg-3.5-el7-release-latest.rpm \
+    && yum install -y --enablerepo=devops osg-build \
+    && yum install -y --enablerepo=osg-empty slurm
+
+RUN cd \
+    && git clone --depth=1 https://github.com/opensciencegrid/gratia-probe \
```

Comment:
Any reason why you're doing this instead of a `COPY` of the repo? Though idk if you can copy `..`

Reply:
Ah, for some reason i thought you wanted whatever was the latest from github.

So do you want to use everything from the checked-out repo that the docker image is built from?

Reply:
Yeah, I think that makes the most sense as a dev tool!

Reply:
```
COPY failed: Forbidden path outside the build context: .. ()
```

Reply:
Ha, that's what I suspected. From the root of the repo, does something like the following work?

```
docker build -t gratia-probe -f test/Dockerfile .
```

https://stackoverflow.com/questions/24537340/docker-adding-a-file-from-a-parent-directory/34300129#34300129